feeadjuster/feeadjuster.py

```diff
@@ -13,6 +13,23 @@
 plugin.update_threshold = 0.05
 
 
+def get_adjusted_percentage(plugin: Plugin, scid: str):
+    """
+    For big channels, there may be a wide range where the liquidity is just okay.
+    Note: if big_enough_liquidity is greater than {total} * 2
+        then percentage is actually {our} / {total}, as it was before
+    """
+    channel = plugin.adj_balances[scid]
+    min_liquidity = min(channel["total"] / 2, int(plugin.big_enough_liquidity))
+    if channel["our"] < min_liquidity:
+        percentage = channel["our"] / min_liquidity / 2
+    elif channel["total"] - channel["our"] < min_liquidity:
+        percentage = (min_liquidity + channel["our"] - channel["total"]) / min_liquidity / 2 + 0.5
+    else:
+        percentage = 0.5
+    return percentage
```

Comment:
We still need testcases that covers the cornercases in here regarding `big_enough_liquidity`.

Reply:
Done.